src/components/shared/map/MapMarkerPopup.tsx

```diff
@@ -9,12 +10,22 @@ import logger from '../../../../logger.config.mjs';
 
 const MapMarkerPopup = ({ seller }: { seller: any }) => {
   const t = useTranslations();
-  const locale = useLocale();
+  const nameRef = useRef<HTMLHeadingElement>(null);
+  const [imageHeight, setImageHeight] = useState(100); // Default height
+
+  // Dynamically adjust the image height based on name height
+  useEffect(() => {
+    if (nameRef.current) {
+      const nameHeight = nameRef.current.offsetHeight;
+      // Reduce image height slightly if name wraps (i.e., height > 22px)
+      setImageHeight(nameHeight > 22 ? 75 : 100);
+    }
+  }, [seller.name]);
 
   const imageUrl =
     seller.image && seller.image.trim() !== ''
       ? seller.image
-      : '/images/logo.svg';
+      : process.env.NEXT_PUBLIC_IMAGE_PLACEHOLDER_URL || '/images/shared/upload.png';
```

Comment:
Another confusing adjustment that may be the result of an unsynced branch with the current dev..? 🤔